src/firebaseInit.js

```diff
@@ -11,15 +11,36 @@ import config from './custom/config'
 // Initialize Firebase
 const firebaseApp = firebase.initializeApp(config.FIREBASE);
 const firestore = firebase.firestore();
+
+function isInIframe () {
+  try {
+    return window.self !== window.top;
+  } catch (e) {
+    return true;
+  }
+}
+
 firestore.settings({ timestampsInSnapshots: true });
 
-firestore.enablePersistence()
+// iFrames may break things for security policies. We found it happens at least in safari.
+// more info in https://firebase.google.com/docs/firestore/manage-data/enable-offline
+if (!isInIframe()) {
+  firestore.enablePersistence()
   .catch(function(err) {
     if (err.code === 'failed-precondition') {
       console.error("Multiple tabs open, persistence can only be enabled in one tab at a a time.");
     } else if (err.code === 'unimplemented') {
       console.error("The current browser does not support all of the features required to enable persistence  ...");
+    } else {
```

Comment:
Lines 34 to 36.
Two else clauses break the code. Should be kept one or another

Reply:
fixed